paasta_tools/cli/cmds/security_check.py

```diff
@@ -47,6 +51,10 @@ def perform_security_check(args):
         return 0
 
     ret_code, output = _run(security_check_command, timeout=300, stream=True)
-    paasta_print("{} exited with code {}".format(security_check_command, ret_code))
+    paasta_print("{} exited with code {}:{}".format(security_check_command, ret_code, output))
+
+    sensu_status = pysensu_yelp.Status.CRITICAL if ret_code else pysensu_yelp.Status.OK
```

Comment:
Is this right? This looks backwards or maybe should be if `ret_code != 0` or something